source/components/text-input/text-input.less

```diff
@@ -9,8 +8,13 @@ textarea {
   padding: 8px;
   width: 100%;
 
+  textarea& {
+    min-height: 5.75rem; // ~92px
```

Comment:
Equivalent to 3 lines of text (with some additional space above and below text).